modules/accredited_representative_portal/spec/models/accredited_representatiive_portal/representative_user_spec.rb

```diff
@@ -91,4 +91,13 @@
       expect(representative_user.flipper_id).to eq(representative_user.email)
     end
   end
+
+  describe '#user_account' do
```

Comment:
@gabezurita Would we like an `InProgressForm`-centric unit test that will protect us from having a breakage if something changes with `InProgressForm`? They may already have such a unit test that tests the scenario where a `User` is provided as input, but there is no unit test that tests when a `RepresentativeUser` is provided as an input.

Our request specs might be sufficient protection from this.

Reply:
I'm okay with testing that intercation at the request spec level and leaving the unit test to cover only the basics!

Reply:
I'll double check that we have the desired protection.